xDrip/Scenes/Home/Home/HomeViewController.swift

```diff
@@ -45,11 +45,17 @@ class HomeViewController: UIViewController, HomeDisplayLogic {
     
     // MARK: IB
     
+    @IBOutlet weak var timeLineSegmentView: TimeFrameSelectionView!
+    
     // MARK: View lifecycle
     
     override func viewDidLoad() {
         super.viewDidLoad()
         doLoad()
+        
+        timeLineSegmentView.delegate = self
+        
+        timeLineSegmentView.config(with: ["1H", "3H", "6H", "12H", "24H"])
```

Comment:
Don't harcode strings, these should be localizable keys. I've added these to localisation document and they now are automatically parsed to `Localizable.strings`.